src/components/AFEpage/index.js

```diff
@@ -1,85 +1,199 @@
+
+            <Typography variant="body1" paragraph>
+              Navgurukul offers a 1-year fully-funded skilling and recruitment
+              program in software engineering that enables them to secure
+              aspirational careers.
+            </Typography>
+          </Grid>
+          <Grid item xs={12} sm={6} md={6} order={{ xs: 1, sm: 2 }}>
+            <img
+              className={classes.IMG}
+              src={require("./asset/Second-img.svg")}
+              alt="Students Img"
+            />
+          </Grid>
+        </Grid>
+
+        <Grid
+          container
+          spacing={!isActive ? 3 : 4}
+          className={classes.Grid_Space}
+        >
+          <Grid item xs={12} sm={6} md={6}>
+            <img
+              className={classes.IMG}
+              src={require("./asset/Second-img.svg")}
+              alt="Students Img"
+            />
+          </Grid>
+          <Grid item xs={12} sm={6} md={6}>
+            <Grid className={classes.Hadings}>
+              <Typography
+                gutterbottom
+                variant="h5"
+                align={isActive ? "center" : "left"}
+              >
+                Our Approach
+              </Typography>
+            </Grid>
+            <Typography variant="body1" paragraph align="left">
+              Navgurukul is working with a two-step approach to make our
+              students CS ready and placed in tech jobs.
+              <List>
+                <ListItem>
+                  <Grid className={classes.AFE_approaches}>
+                    <FiberManualRecordIcon
+                      fontSize="10px"
+                      className={classes.Fiber_space}
+                    />
+                    Accessibility and readiness program: Meraki
+                    <ListItemText className={classes.Approaches_text}>
+                      Age group: 13+ girls/ women
+                    </ListItemText>
+                  </Grid>
+                </ListItem>
+                <ListItem>
+                  <Grid className={classes.AFE_approaches}>
+                    <FiberManualRecordIcon
+                      className={classes.Fiber_space}
+                      fontSize="Small"
+                    />
+                    Skilling and job placement program: Residential Program
+                    <ListItemText className={classes.Approaches_text}>
+                      Age group 18 - 29 years
+                    </ListItemText>
+                  </Grid>
+                </ListItem>
+              </List>
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid spacing={8} align="center" className={classes.Ng_AFE}>
+          {!isActive ? (
+            <Typography gutterbottom variant="h5">
+              NavGurukul 🤝 Amazon Future Engineer
+            </Typography>
+          ) : (
+            <Typography gutterbottom variant="h5">
+              NavGurukul 🤝 AFE
+            </Typography>
+          )}
```

Comment:
Using acronyms for small screens might occur in other places.  As such, you might want to create a new Component for this (Using the `isSmallScreen` from before):

```
export default function SmallScreenAbbreviate(props) {
  const text = props.text || "";
  const abbrText = text.split(" ").map(word => word[0].toUpperCase()).join("")
  return isSmallScreen() ? <abbr title = {text}>{abbrText}</abbr> : text;
}
```
 
```suggestion
          <Typography gutterbottom variant="h5">
            NavGurukul 🤝 <SmallScreenAbbreviate text = "Amazon Future Engineer" /> 
          </Typography>
```